pallets/pallet-nft/src/lib.rs

```diff
@@ -75,14 +124,32 @@ pub mod module {
 		NftTransferred { from: T::AccountId, to: T::AccountId, nft_id: NftId },
 		/// Emitted when a new NFT is requested to be minted, pending audit.
 		NFTPending { nft_id: NftId, file_name: Vec<u8> },
-		/// Emitted when an NFT creation is rejected by the auditor.
+		/// Emitted when an NFT creation is rejected by the auditor or the receiver rejected the
```

Comment:
Should use a different event for when POD is rejected.
It represents a different scenario from when NFT MInt is rejected.